recipes/crunch/all/CMakeLists.txt

```diff
@@ -0,0 +1,235 @@
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
Bump also to 3.1?

Reply:
I don't think it's necessary. It doesn't use features from CMake 3.0 and later.

Reply:
Yes, only necessary for the `test_package`

There was a Conan discussion about the minimum supported CMake version. If Conan raises the version, we can do it here as well. Otherwise, we only should do it for concrete reasons.